### PR TITLE
[E2E Testing] Publish customize oap image

### PR DESCRIPTION
**Pull request by hailin0**

### CODE REVIEW

**Comment by wu-sheng** [comment]
Why don't you build locally or during the CI, and install locally only? The ASF snapshot repo is not stable, and not immutable.

**Comment by wu-sheng** [comment]
In that case, you don't need to deploy. You just need a local `mvn install`, and build the project based on snapshot version.

**Comment by wu-sheng** [comment]
How about skipping the publish step. Just build the container in the CI env, and run the following testing?
The main repo e2e is using one task to build binary tar and image, then share with all following CI tasks.

If we run publishing, this will become very tricky, when should build it, and who should use it after PR merged?

**Comment by wu-sheng** [comment]
@hanahmily This is very tricky. Is this acceptable?
This is a one time work now.

**Comment by wu-sheng** [comment]
As we had discussed, this is a one time temp publish, let's close this for now as it is published successfully.

**Comment by wu-sheng** [comment]
@hanahmily @lujiajing1126 Please check this.

As properties TTL feature in 0.5 is still not added, should we wait for that to do a release? Or should we run a RC release?

**Comment by hanahmily** [comment]
> @hanahmily @lujiajing1126 Please check this.
> 
> 
> 
> As properties TTL feature in 0.5 is still not added, should we wait for that to do a release? Or should we run a RC release?
> 
> 

We will wait for the TTL to do a release.

File: src/main/java/org/apache/skywalking/banyandb/v1/client/StreamBulkWriteProcessor.java
Comment (lujiajing1126) [inline src/main/java/org/apache/skywalking/banyandb/v1/client/StreamBulkWriteProcessor.java:81]: What would happen if we throw an exception here?

**Review by lujiajing1126 — CHANGES_REQUESTED** [review]

File: src/main/java/org/apache/skywalking/banyandb/v1/client/StreamBulkWriteProcessor.java
Comment (wu-sheng) [reply src/main/java/org/apache/skywalking/banyandb/v1/client/StreamBulkWriteProcessor.java:81]: I think we usually through banyandb exception rather than runtime exception, right?

**Pull request closed without merging**
